### ISSUE TITLE
Get updates from Github releases

### ISSUE THREAD

**Issue by bacca87** [ravibpatel/AutoUpdater.NET#296]
Using the Github api you can download a json with the latest release:

`https://api.github.com/repos/:owner/:repo/releases/latest`

ex: https://api.github.com/repos/ravibpatel/AutoUpdater.NET/releases/latest

but the event **AutoUpdaterOnParseUpdateInfoEvent** is never fired.
can someone help me to understand why this is not working?

My code:
```
public void Main()
{
    AutoUpdater.ParseUpdateInfoEvent += AutoUpdaterOnParseUpdateInfoEvent;
    AutoUpdater.Start("https://api.github.com/repos/ravibpatel/AutoUpdater.NET/releases/latest");
}

private void AutoUpdaterOnParseUpdateInfoEvent(ParseUpdateInfoEventArgs args)
{
    dynamic json = JsonConvert.DeserializeObject(args.RemoteData);
    // do stuff
}
```

Thx in advice

**Comment by ravibpatel** [comment]
Can you try the new version from [here](https://ci.appveyor.com/api/buildjobs/omng4wmuvv4dm20b/artifacts/AutoUpdater.NET-1.5.4.zip). It should fix the issue.

**Comment by bacca87** [comment]
I'm already using this version and doesn't work!

**Comment by bacca87** [comment]
I found the issue by myself and i made a pullrequest in order to fix that.

**Comment by ravibpatel** [comment]
Hi updated it but didn't updated the assembly version. Once you test it out I will do that.

**Comment by bacca87** [comment]
ok i checked your changes. 
In my opinion could be better to add the possibility to set a custom useragent, if the custom user agent is not setted, you can use the default one.
This is useful in order to keep aligned the app user agent with the auto updater one.
Please let me know what you do think!

**Comment by ravibpatel** [comment]
I merged your changes. You can try it out from [here](https://ci.appveyor.com/api/buildjobs/31ed2clduvq13i6t/artifacts/AutoUpdater.NET-1.5.4.zip).

**Comment by bacca87** [comment]
any plan for your next release? i need urgently this update and i need to understand if i can wait or is better to build this version by myself....

**Comment by ravibpatel** [comment]
Did you test out the latest changes I made? If it is working fine I can release the new version.

**Comment by bacca87** [comment]
Yes, i did and works as expected.
I've also fixed the italian translations #299

**Comment by ravibpatel** [comment]
I updated the NuGet package to v1.5.5.
